requirements.txt

```diff
@@ -1,4 +1,6 @@
-# This can not be installed from pip, on Fedora you need to install rpm-python{,3}
+# This can not be installed from pip, on Fedora you need to install
+# python{2,3}-rpm RPM including rpm-python pip package.
```

Comment:
Squash merging is disabled for this project. To be honest I'm not sure why, but anyway it would be better if you rebased your branch to master and squashed the two commits yourself. Note that there is a typo in the title (`Imporve`), and also the commit message
`Update RPM package name python-rpm to python2-rpm in requirements.txt`
should most likely be modified like this:
`Update RPM package name rpm-python{,3} to python{2,3}-rpm in requirements.txt`